.github/pull_request_template.md

```diff
@@ -1,18 +1,86 @@
-Describe your changes here.
+## Summary
 
-Closes # (type the issue number after # if applicable; otherwise remove this line)
+<!--
+A few sentences describing the changes being proposed in this pull request.
+-->
 
-### Screenshots
+## List of notable changes
```

Comment:
Another one of framing this could be as a "changelog", 

```
#### Changelog

**New**

- {{new thing}}

**Changed**

- {{changed thing}}

**Removed**

- {{removed thing}}
```

This could help to breakdown the impact of the PR as it relates to the package being updated.